### ISSUE TITLE
How Working??

### ISSUE THREAD

**Issue by bitandalib** [mustafauysal/bp-stories#2]
I installed in my site. but I can't work with that :(

**Comment by mustafauysal** [comment]
Hey @bitandalib 

It's not working for now. It was sort of demo, I hope to make it happen someday.

**Comment by derintolu** [comment]
Oh man I wish this was real!! I know a few people who would pay for something like this.

**Comment by Triex** [comment]
Agreed ^ also super keen on this. May create a fork & see if I can work on it!

**Comment by mustafauysal** [comment]
Honestly, I thought this to make a premium plugin. However, the audience is quite a niche, and maintaining long-term is not so easy. It's still a draft project in my mind, but I don't think that I can work on this anytime soon :(

That would be great to see a fork.
